hail/python/hail/docs/cheatsheets.rst

```diff
@@ -5,8 +5,20 @@ Cheatsheets
 =============
 
 .. note::
-    Hail's cheatsheets are in their early stages. We welcome suggestions
+    Hail's cheatsheets are relatively new. We welcome suggestions
     for additional cheatsheets, as well as feedback about our documentation. If
     you'd like to add a cheatsheet to the documentation, make a pull request!
 
-`Hail Tables Cheatsheet <https://github.com/hail-is/hail/raw/master/hail/cheatsheets/hail_tables_cheat_sheet.pdf>`_
\ No newline at end of file
+Hail Tables Cheatsheet
+----------------------
+
+`Download Tables Cheatsheet <https://github.com/hail-is/hail/raw/master/hail/cheatsheets/hail_tables_cheat_sheet.pdf>`_
+
+`View Tables Cheatsheet on Github <https://github.com/hail-is/hail/blob/master/hail/cheatsheets/hail_tables_cheat_sheet.pdf>`_
```

Comment:
I think it's probably enough to have one link. When you visit the blob link, you get access.

Also, same issue as before: we should version these documents, and host them alongside docs, so that we can have API version pinned to release. I think you opened an issue for this at one point, but if not please add one.